translation.gemspec

```diff
@@ -4,7 +4,7 @@ Gem::Specification.new do |s|
   s.description      = 'Localize your app using either t(".keys") or _("source text") and type "rake translation:sync" to synchronize with your translators on Translation.io.'
   s.homepage         = 'https://translation.io'
   s.email            = 'contact@translation.io'
-  s.version          = '1.16'
+  s.version          = '1.16.8'
```

Comment:
plz ignore.
was testing on a docker environment and needed to update the version to get it to `bundle update`, will change back.